app/services/email_alert_api/subscriber_list_updater.rb

```diff
@@ -0,0 +1,85 @@
+module EmailAlertApi
+  class SubscriberListUpdater
+    include ParamsFormatter
+
+    def self.call(...)
+      new(...).handle
+    end
+
+    attr_reader :item, :successor
+
+    def initialize(item:, successor:)
```

Comment:
Assuming item here is `content_item`, is it worth calling it as such for consistency with other files?

Reply:
Ah damn, I should have done that in https://github.com/alphagov/collections-publisher/pull/1965 

Yes, I'll update, thanks

Reply:
https://github.com/alphagov/collections-publisher/pull/1960/commits/d10b1a8f1a427d39e78504fdd456a43768449f4a